pkg/mesos/client.go

```diff
@@ -22,7 +22,29 @@ func NewClient(baseClient *httpclient.Client) *Client {
 	}
 }
 
-// Masters is returning the mesos masters of the connected cluster.
+// HTTP returns the HTTP client of the Mesos client.
+func (c *Client) HTTP() *httpclient.Client {
+	return c.http
+}
+
+// Leaders returns the mesos leaders of the connected cluster.
+func (c *Client) Leaders() ([]Master, error) {
```

Comment:
Should this function maybe only return one Master? I know it is possible to get multiple but is that ever useful? Should Mesos ever have multiple leaders? Otherwise I would suggest to already check here if multiple leaders are returned and if so throw an error. Unless it is useful to have multiple leaders, this would save checking for that case when this function is called.